contracts/vesting/TerminableVestingWallet.sol

```diff
@@ -0,0 +1,51 @@
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts/finance/VestingWallet.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+
+contract TerminableVestingWallet is VestingWallet, Ownable {
+  bool public isTerminated;
+  uint256 public terminationTimestamp;
```

Comment:
Do we need `isTerminated` at all if we have `terminationTimestamp`?

We could set this timestamp to max value or store here a difference between vesting end and termination timestamp to default it to 0